frontends/PyCDE/test/pycde_types.py

```diff
@@ -1,6 +1,15 @@
 # RUN: %PYTHON% %s 2>&1 | FileCheck %s
 
 from pycde import dim, types
+import sys
+
+# CHECK: [('foo', Type(i1)), ('bar', Type(i13))]
+# CHECK: i1
+st1 = types.struct({"foo": types.i1, "bar": types.i13})
+fields = st1.get_fields()
+sys.stderr.write(str(fields) + "\n")
```

Comment:
I assume there's a good reason why not just call `print`... I'm curious what that is.

Reply:
The `dump()` calls output to stderr and the ordering of stderr lines vs stdout lines in pipes isn't well defined (even with flushes) so outputting everything to stderr is the easiest way to ensure that filecheck sees the lines in the correct order.

Reply:
Ah, I see. If the order is non-deterministic, and doesn't matter, it may be possible to use CHECK-DAG in this situation? Not a big deal either way, having the test output be deterministic sounds good to me :)

Reply:
Yeah, I prefer it to be deterministic and this wasn't much of a compromise.

Reply:
FYI not sure if you saw this, but after merging both our PRs, I found that I needed a flush (at least on my Ubuntu system) https://github.com/llvm/circt/commit/70e7402caf98a11f99de8739ec19901482c2933c#diff-e75cddaa2cef70fd4c3cee85dc25d237929e8029f0c73c1db59c07d534dfeb92R11